optuna/storages/_rdb/storage.py

```diff
@@ -347,23 +359,33 @@ def get_all_study_summaries(self) -> List[StudySummary]:
         study_summary_stmt = session.query(
             models.StudyModel.study_id,
             models.StudyModel.study_name,
-            models.StudyModel.direction,
             summarized_trial.c.datetime_start,
             functions.coalesce(summarized_trial.c.n_trial, 0).label("n_trial"),
         ).select_from(orm.outerjoin(models.StudyModel, summarized_trial))
 
         study_summary = study_summary_stmt.all()
         study_summaries = []
         for study in study_summary:
+            directions = [
+                d.direction for d in models.StudyDirectionModel.where_study(study, session)
+            ]
             best_trial: Optional[models.TrialModel] = None
             try:
-                if study.direction == StudyDirection.MAXIMIZE:
+                if len(directions) > 1:
+                    raise ValueError
```

Comment:
[Optional] I think these lines will be updated when we integrate multi-objective optimization. It may be kind for readers to add a code comment or an error message about it.